kong/plugins/acl/handler.lua

```diff
@@ -1,47 +1,76 @@
-local singletons = require "kong.singletons"
-local BasePlugin = require "kong.plugins.base_plugin"
-local cache = require "kong.tools.database_cache"
-local responses = require "kong.tools.responses"
-local utils = require "kong.tools.utils"
-local constants = require "kong.constants"
+local singletons   = require "kong.singletons"
+local BasePlugin   = require "kong.plugins.base_plugin"
+local cache        = require "kong.tools.database_cache"
+local responses    = require "kong.tools.responses"
+local utils        = require "kong.tools.utils"
+local constants    = require "kong.constants"
+
 
 local table_insert = table.insert
 local table_concat = table.concat
-local ipairs = ipairs
-local empty = {}
+local ngx          = ngx
+local set_header   = ngx.req.set_header
+local ipairs       = ipairs
+local type         = type
+local next         = next
+local empty        = {}
+
 
 local ACLHandler = BasePlugin:extend()
 
+
```

Comment:
It seems like this PR has many style changes. As we've agreed upon by the past, could we limit the scope of the changes in the PRs? Or if we want those changes in the same PR, could they be in separate commits at least? 

- `feat(acl) work with consumer`
- `style(acl) update to new style`
- `perf(acl) minor improvements`